app/controllers/api/watches/toggle_controller.rb

```diff
@@ -9,11 +9,15 @@ def index
   end
 
   def create
-    @watch = Watch.create!(
-      user: current_user,
-      watchable: watchable
-    )
-    render json: @watch
+    if Watch.exists?(user_id: current_user.id, watchable_id: params[:watchable_id], watchable_type: params[:watchable_type])
+      render json: { message: "この#{watchable.class.model_name.human}はWatch済です。" }, status: :unprocessable_entity
```

Comment:
```suggestion
    is_exists = Watch.exists?(
      user_id: current_user.id, 
      watchable_id: params[:watchable_id], 
      watchable_type: params[:watchable_type]
    )
    if is_exists
      message = "この#{watchable.class.model_name.human}はWatch済です。"
      render json: { message: message }, status: :unprocessable_entity
```

横に長過ぎるので改行などを使った方がいいかなと思いました。